src/applications/facility-locator/components/FacilityTypeDropdown.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import { kebabCase } from 'lodash/fp';
+import classNames from 'classnames';
+import Downshift from 'downshift';
+
+import { facilityTypes } from '../config';
+import { keyMap } from '../utils/helpers';
+
+const FACILITY_OPTIONS = ['all', 'health', 'benefits', 'cemetery', 'vet_center'];
+
+const facilityOptionClasses = (item, selected) => classNames(
+  'dropdown-option',
+  { selected },
+  { 'icon-option': item && item !== 'all' },
+  { [`${kebabCase(item)}-icon`]: item && item !== 'all' }
+);
+
+const itemToString = (item) => facilityTypes[item] || 'All Facilities';
+
+const focusDropdown = () => document.querySelector('#facility-dropdown-toggle').focus();
```

Comment:
Is using refs of interest here as an alternative to querying the DOM?

Reply:
Thought about using a `ref`, but figured it would be less readable. Querying by element id is efficient enough, I think.